app/services/aapb/batch_ingest/pbcore_xml_mapper.rb

```diff
@@ -44,8 +44,16 @@ def prepare_annotations(annotations)
 
       def find_annotation_type_id(type)
         type_id = Annotation.find_annotation_type_id(type)
-        return type_id if type_id.present?
-        raise "annotation_type not registered with the AnnotationTypesService: #{type}."
+
+        if ENV['SETTINGS__BULKRAX__ENABLED'] == 'true'
+          type_id
+        else
+          if type_id.present?
+            type_id
+          else
+            raise "annotation_type not registered with the AnnotationTypesService: #{type}."
+          end
+        end
```

Comment:
this was one of those validation errors that Tim keeps talking about. A lot of the example data sets would fail on this error, but the records seem to be otherwise valid. I thought that the goal was to let the ams1 stuff in close to as is at this point. This may need requirement discussion?